src/foam/u2/wizard/XORMinMaxWAO.js

```diff
@@ -129,22 +131,22 @@
       }
 
 
-      if ( this.loadIntoPropertyName ){
+      if ( this.loadIntoPath ){
 
         if ( ! wizardlet.data ){
           wizardlet.data = this.of.create({}, this);
         }
 
-        wizardlet.data[this.loadIntoPropertyName] = clonedWizardletData;
+        this.loadIntoPath.f(wizardlet.data, clonedSelectedWizardletData);
```

Comment:
I don't think `.f(...)` will work for this, but I added `$set` to handle it
```suggestion
        this.loadIntoPath$set(wizardlet.data, clonedSelectedWizardletData);
```